app/slack_notifiers/application_slack_notifier.rb

```diff
@@ -6,7 +6,7 @@ class ApplicationSlackNotifier < SlackNotifier
   private
 
   def default_url_options
-    { host:, port: ENV.fetch("PORT") }
+    { host:, port: ENV["PORT"] }
```

Comment:
If we can use this without `port` do we even need it as an arguments?

Reply:
We need it for development, but I can clean it up post fix.